newsletters/2023-09.md

```diff
@@ -0,0 +1,69 @@
+---
+tags: obf
+---
+
+_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/FIX THIS LINK)._
+
+_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._
+
+# OBF Newsletter September 2023
+
+TL;DR: In this newsletter we announce the OBF's upcoming public board meeting; provide a summary of BOSC 2023 in Lyon; share news about the OBF event fellowship; and announce an opening for an OBF admin for the 2024 Google Summer of Code. 
+
+In longer form: 
+
+## OBF News
+
+### Upcoming public board meeting
+
+The OBF is having its next public board meeting on XXX. On our agenda is XXX. You can find more details including how to join us on the OBF Blog [add link]. 
+
+### BOSC news
+
+The 24th annual Bioinformatics Open Source Conference (BOSC 2023) took place in July as part of ISMB/ECCB 2023 in Lyon, France, and online. As in prior years, BOSC was preceded by a _CollaborationFest_, a collaborative work event (including but not limited to hacking). This year CoFest was hosted by Jérémy Just at the nearby _École Normale Supérieure de Lyon_, which sponsored space and infrastructure for the 25 in-person participants as well as the numerous online participants. 
+
+In addition to 43 talks and nearly 50 posters, BOSC itself included two keynotes – by Sara El-Gebali, who spoke about _"A New Odyssey: Pioneering the Future of Scientific Progress Through Open Collaboration_ – and by Joseph Yracheta – who spoke about _"The Dissonance between Scientific Altruism & Capitalist Extraction: The Zero Trust and Federated Data Sovereignty Solution”_. The conference closed out with a panel on _Open & Ethical Data Sharing_. 
+
+You can [read the full report about the conference on the OBF blog](https://www.open-bio.org/2023/08/14/bosc-2023-report/).
+
+### OBF Travel Fellowships
+
+Next round: December 1, 2023
+Last round: August 1, 2023
+
+The current application period for the OBF Event Fellowship lasts until December 1, 2023. So there is still some [time to get your application submitted](https://docs.google.com/forms/d/e/1FAIpQLScMayX_XwTLxc8MiYgQOHIILJNjxF0gL75TUsbaO17h5UuUeA/viewform).
+
+These fellowships can be used to cover registration fees for relevant events related to open source bioinformatics or open science (in-person or virtual), and potentially additional expenses such as travel, lodging, childcare for the duration of the event, small hardware (e.g. a headset or webcam), or high-speed internet. You can read more about [applying for an OBF Event Fellowship](https://www.open-bio.org/event-awards/#fellowships-applications), and [the experiences of past awardees](https://www.open-bio.org/category/travel-fellowship/event-fellowship/).
```

Comment:
Add

## Sponsors needed!

OBF Event Fellowships, which seek to increase diverse representation at relevant events, are funded by sponsorships. Most of these come through [BOSC](https://www.open-bio.org/events/sponsors/), but we also welcome donations to the OBF. Please contact us at [] if you are interested in discussing sponsorship possibilities.

Reply:
Added as well :) Do we have someone that should be the point person to link to?

Reply:
i don't know. we need someone in OBF to take responsibility for that. (not me!)

Reply:
Could default to "Please contact the board if ..." (need not be a email link), but I'm inclined to drop this text for now. We don't have anything setup to thank non-BOSC sponsors beyond a one off blog post like we had for the NESCent funds.

Reply:
ok, we can take this out, but this is something we need to work on.

Reply:
Okay, I'll remove that part!